pkg/tui/page/chat/chat.go

```diff
@@ -430,6 +434,31 @@ func (p *chatPage) switchFocus() {
 	}
 }
 
+// handleSlashCommand checks if the content is a slash command and handles it
+// Returns (true, cmd) if command was handled, (false, nil) otherwise
+func (p *chatPage) handleSlashCommand(content string) (bool, tea.Cmd) {
```

Comment:
These commands already exist in the tui.go 

https://github.com/docker/cagent/blob/605590918ce98a5cf6053afacc0e965286def382/pkg/tui/tui.go#L293-L333

We should reuse them